app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -7,16 +7,12 @@ class PermissionTemplateForm
       self.model_class = PermissionTemplate
       self.terms = []
       delegate :access_grants, :access_grants_attributes=, :release_date, :release_period, :visibility, to: :model
-      delegate :available_workflows, :active_workflow, :source_model, to: :model
-
-      # @return [#to_s] the primary key of the associated admin_set or collection
-      # def source_id (because you might come looking for this method)
-      delegate :id, to: :source_model, prefix: :source
+      delegate :available_workflows, :active_workflow, :source, :source_id, to: :model
```

Comment:
The previous delegate to get the `id` from `source_model` resulted in an unnecessary population of the `source_model` just to get the `id` which is already available from the template using `source_id`.  So this removes the original delation and adds a direct delegate to `:source_id`.

NOTE: There is still access to the `source_model` in the template even though the delegation was removed.  There is a method named `#source_model` defined later in this class that include a note recommending use of `#source` unless the ActiveFedora model is really needed.